src/test/mir-opt/const_prop/bad_op_div_by_zero/rustc.main.ConstProp.diff

```diff
@@ -46,6 +46,12 @@
 +                                          // + span: $DIR/bad_op_div_by_zero.rs:5:14: 5:19
 +                                          // + literal: Const { ty: bool, val: Value(Scalar(0x01)) }
           assert(!move _4, "attempt to divide by zero") -> bb1; // scope 1 at $DIR/bad_op_div_by_zero.rs:5:14: 5:19
+                                           // ty::Const
+                                           // + ty: i32
+                                           // + val: Value(Scalar(0x00000001))
+                                           // mir::Constant
+                                           // + span: $DIR/bad_op_div_by_zero.rs:5:14: 5:15
+                                           // + literal: Const { ty: i32, val: Value(Scalar(0x00000001)) }
```

Comment:
So much additional useless debug output...^^
And this is rather confusing in fact because it is very unclear why it is even printed (it's the hidden operand of `assert` above).

Reply:
This is now represented in MIR